### PR TITLE
Add notification settings screen

### PR DESCRIPTION
**Pull request by maurizi**
## Overview

Adds a new tab to the user settings screen allowing them to configure webhooks for notifications.

Connects #1991 

## Demo

notification-settings

## Notes

I looked at the `UserProfile` component also on the settings page for inspiration on what libraries to use / how to style the page - if that wasn't the best example to draw from it shouldn't be too hard to change that.

## Testing Instructions

* `scripts/update`
* Use the new Notifications tab of the settings page to add, update, test and delete some webhooks

## Checklist

- [x] `fixup!` commits have been squashed
- [ ] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- [ ] If this PR applies to both OAR and OGR a companion PR has been created

### CODE REVIEW

**Comment by jwalgran** [comment]
I ran into an exception when trying to apply migrations after having `ogr/develop` checked out and fully migrated

```
django.db.utils.ProgrammingError: relation "api_event" already exists
```

I think this may be because the Event migration was renamed from 0098 to 0099

Screen Shot 2022-08-02 at 2 16 28 PM

**Comment by maurizi** [comment]
> I ran into an exception when trying to apply migrations after having ogr/develop checked out and fully migrated

Yes, I renumbered the wrong migration after rebasing, fixed in https://github.com/open-apparel-registry/open-apparel-registry/pull/2013/commits/4d9bfe029303dea3f5db315f4d6a95705383b5ef

**Review by jwalgran — APPROVED** [review]
Excellent implementation.  I confirmed the CRUD actions and the test button.

Screen Shot 2022-08-03 at 11 01 12 AM

File: src/app/src/actions/webhooks.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+import apiRequest from '../util/apiRequest';
+
+import {
+    makeUserWebhooksURL,
+    makeUserWebhookURL,
+    logErrorAndDispatchFailure,
+} from '../util/util';
+
+export const startFetchUserWebhooks = createAction('START_FETCH_WEBHOOKS');
```
[the remark this answers is not in the document]
Comment (maurizi) [reply src/app/src/actions/webhooks.js:11]: This was a pretty simple find/replace (VSCode even formatted the code w/ `prettier` for me 😄 )

**Pull request merged**